scripts/test/lib_subctl_gather_test.sh

```diff
@@ -61,7 +61,7 @@ function validate_pod_log_files() {
   read -ra pod_names_array <<< "$pod_names"
 
   for pod_name in "${pod_names_array[@]}"; do
-    file=$gather_out_dir/${cluster}_$pod_name.log
+    file=$gather_out_dir/${cluster}/${cluster}_$pod_name.log
```

Comment:
```suggestion
    file="${gather_out_dir}/${cluster}/${pod_name}.log"
```

Let's instead omit cluster name from the log file name in https://github.com/submariner-io/subctl/blob/devel/internal/gather/logs.go#L105